src/OpenLoco/Windows/Vehicle.cpp

```diff
@@ -2596,36 +2622,24 @@ namespace OpenLoco::Ui::Vehicle
         static void getScrollSize(Ui::window* const self, const uint32_t scrollIndex, uint16_t* const width, uint16_t* const height)
         {
             auto head = Common::getVehicle(self);
-
-            // Space for the end of orders and express/local item
-            *height = 10 * 2;
-
             auto table = getOrderTable(head);
-            *height += 10 * std::distance(table.begin(), table.end());
+            *height = 10 * std::distance(table.begin(), table.end());
```

Comment:
You've removed the end of orders as well

Reply:
An ironic regression. :) Thanks!